package.json

```diff
@@ -62,5 +62,115 @@
     }
-  }
+  },
+  "files": [
+    "/es/h.d.ts",
+    "/es/h.js",
+    "/es/h.js.map",
+    "/es/helpers/attachto.d.ts",
+    "/es/helpers/attachto.js",
+    "/es/helpers/attachto.js.map",
+    "/es/hooks.d.ts",
+    "/es/hooks.js",
+    "/es/hooks.js.map",
+    "/es/htmldomapi.d.ts",
+    "/es/htmldomapi.js",
+    "/es/htmldomapi.js.map",
+    "/es/is.d.ts",
+    "/es/is.js",
+    "/es/is.js.map",
+    "/es/modules/attributes.d.ts",
+    "/es/modules/attributes.js",
+    "/es/modules/attributes.js.map",
+    "/es/modules/class.d.ts",
+    "/es/modules/class.js",
+    "/es/modules/class.js.map",
+    "/es/modules/dataset.d.ts",
+    "/es/modules/dataset.js",
+    "/es/modules/dataset.js.map",
+    "/es/modules/eventlisteners.d.ts",
+    "/es/modules/eventlisteners.js",
+    "/es/modules/eventlisteners.js.map",
+    "/es/modules/hero.d.ts",
+    "/es/modules/hero.js",
+    "/es/modules/hero.js.map",
+    "/es/modules/module.d.ts",
+    "/es/modules/module.js",
+    "/es/modules/module.js.map",
+    "/es/modules/props.d.ts",
+    "/es/modules/props.js",
+    "/es/modules/props.js.map",
+    "/es/modules/style.d.ts",
+    "/es/modules/style.js",
+    "/es/modules/style.js.map",
+    "/es/snabbdom.bundle.d.ts",
+    "/es/snabbdom.bundle.js",
+    "/es/snabbdom.bundle.js.map",
+    "/es/snabbdom.d.ts",
+    "/es/snabbdom.js",
+    "/es/snabbdom.js.map",
+    "/es/thunk.d.ts",
+    "/es/thunk.js",
+    "/es/thunk.js.map",
+    "/es/tovnode.d.ts",
+    "/es/tovnode.js",
+    "/es/tovnode.js.map",
+    "/es/vnode.d.ts",
+    "/es/vnode.js",
+    "/es/vnode.js.map",
+    "/modules/attributes.d.ts",
+    "/modules/attributes.js",
+    "/modules/attributes.js.map",
+    "/modules/class.d.ts",
+    "/modules/class.js",
+    "/modules/class.js.map",
+    "/modules/dataset.d.ts",
+    "/modules/dataset.js",
+    "/modules/dataset.js.map",
+    "/modules/eventlisteners.d.ts",
+    "/modules/eventlisteners.js",
+    "/modules/eventlisteners.js.map",
+    "/modules/hero.d.ts",
+    "/modules/hero.js",
+    "/modules/hero.js.map",
+    "/modules/module.d.ts",
+    "/modules/module.js",
+    "/modules/module.js.map",
+    "/modules/props.d.ts",
+    "/modules/props.js",
+    "/modules/props.js.map",
+    "/modules/style.d.ts",
+    "/modules/style.js",
+    "/modules/style.js.map",
+    "/helpers/attachto.d.ts",
+    "/helpers/attachto.js",
+    "/helpers/attachto.js.map",
+    "/h.d.ts",
+    "/h.js",
+    "/h.js.map",
+    "/hooks.d.ts",
+    "/hooks.js",
+    "/hooks.js.map",
+    "/htmldomapi.d.ts",
+    "/htmldomapi.js",
+    "/htmldomapi.js.map",
+    "/is.d.ts",
+    "/is.js",
+    "/is.js.map",
+    "/snabbdom.bundle.d.ts",
+    "/snabbdom.bundle.js",
+    "/snabbdom.bundle.js.map",
+    "/snabbdom.d.ts",
+    "/snabbdom.js",
+    "/snabbdom.js.map",
+    "/thunk.d.ts",
+    "/thunk.js",
+    "/thunk.js.map",
+    "/tovnode.d.ts",
+    "/tovnode.js",
+    "/tovnode.js.map",
+    "/vnode.d.ts",
+    "/vnode.js",
+    "/vnode.js.map"
+  ]
```

Comment:
The `files` field is considered better practice than a `.npmignore` because it prevents accidental publishing of private files, for one.